app/views/rating_questions/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form"><%= form_with model: @rating_question, url: rating_questions_path,local: true do |f| %>
```

Comment:
This form component is pretty much the same as the `edit.html.erb`. We could probably move it into a partial.

Reply:
will look into that.